pysmiles/smiles_helper.py

```diff
@@ -460,6 +458,10 @@ def mark_aromatic_atoms(mol, atoms=None, prefill_valence=False):
         The molecule.
     atoms: collections.abc.Iterable
         The atoms to act on. Will still analyse the full molecule.
+    correct_aromatic: bool
+        If the falg is set then all nodes are considered
+        in the kekulization process otherwise only aromatic
+        nodes are considered.
```

Comment:
This option kind of conflicts with the `atoms` option. How hard is it to use one or the other?

Reply:
so this option I didn't understand to be honest; it says it will only work on a subset of nodes but work on the whole molecule? In the end would be fne to use either one. Just that the selection of aromatic nodes would move to outside the mark_aromatic function.